spec/system/placements/schools/mentors/remove_a_mentor_spec.rb

```diff
@@ -0,0 +1,114 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Schools / Mentors / Remove a mentor", type: :system, service: :placements do
+  let(:school) { create(:placements_school, name: "School 1") }
+  let(:mentor_1) do
+    create(:placements_mentor,
+           first_name: "John",
+           last_name: "Doe",
+           trn: "1234567")
+  end
+  let(:mentor_2) do
+    create(:placements_mentor,
+           first_name: "Agatha",
+           last_name: "Christie",
+           trn: "2222222")
```

Comment:
I wonder if you really need to set the trn here? It's not used explicitly in the test and the factory will generate a valid one.

Reply:
resolved in [1f2f2be](https://github.com/DFE-Digital/itt-mentor-services/pull/295/commits/1f2f2be476f95a70eedc3b7c87a1fd06fb170e92)